docs/src/reference/compilation.rst

```diff
@@ -237,6 +237,80 @@ Just as an example, this adds ``mylib`` as library to every extension::
     then the argument to ``create_extension`` must be pickleable.
     In particular, it cannot be a lambda function.
 
+Cythonize arguments
+-------------------
+
+The function :func:`cythonize` can take extra arguments which will allow you to
+customize your build.
+
+.. py:function:: cythonize(module_list, \
+                           exclude=None, \
+                           nthreads=0, \
+                           aliases=None, \
+                           quiet=False, \
+                           force=False, \
+                           language=None, \
+                           exclude_failures=False, \
+                           **options)
+
+    Compile a set of source modules into C/C++ files and return a list of distutils
+    Extension objects for them.
+
+    :param module_list: As module list, pass either a glob pattern, a list of glob
+                        patterns or a list of Extension objects.  The latter
+                        allows you to configure the extensions separately
+                        through the normal distutils options.
```

Comment:
It's even cooler than that! :)
You can also pass `Extension` objects that have `glob` patterns as their `sources`. Then, `cythonize` will resolve the pattern and create a copy of the  `Extension` for every matching file.

Reply:
Fixed next line.